### PR TITLE
Splitting human life into organs + removes the advanced operating table

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
Part 2 out of 3 of the brainmed port.

To-do:

- [x] Wait for arteries to get merged lmao

- [x] Get the rest of the TODOs done

- [x] Maybe some extra stuff?

- [x] Testing

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!wip

**Comment by NonQueueingMatt** [comment]
!review

**Comment by NonQueueingMatt** [comment]
When reviewing please tell me if there's a better way to handle the loss of lungs

Right now that's handled in handle breath

File: code/modules/organs/organ.dm [hunk 1]
```diff
@@ -363,7 +363,7 @@
 	owner.update_action_buttons()
 	owner = null
 
-	if(!owner.isonlifesupport())
+	if(owner && owner.isonlifesupport())
```
Comment (MarinaGryphon) [inline code/modules/organs/organ.dm]:
```suggestion
	if(owner && !owner.isonlifesupport())
```

**Comment by Alberyk** [comment]
Stamina is not recovering.

**Comment by NonQueueingMatt** [comment]
stamina is fixed

**Pull request merged**